circuit_maintenance_parser/data.py

```diff
@@ -0,0 +1,47 @@
+"""Definition of Data classes."""
+import logging
+
+from collections import namedtuple
+from typing import List
+
+import email
+from pydantic import BaseModel, Extra
+
+
+logger = logging.getLogger(__name__)
+
+
+DataPart = namedtuple("DataPart", "type content")
+
+
+class NotificationData(BaseModel, extra=Extra.forbid):
+    """Base class for Notificaiton Data types."""
+
+    _data_parts: List[DataPart] = []
+
+    def get_data_parts(self) -> List[DataPart]:
+        """Return an Iterable of Data objects from each NotificationData class."""
+        return self._data_parts
```

Comment:
Why does `_data_parts` need to be a private attribute with a dedicated getter function? Why not just a public instance attribute that can be accessed directly?

Reply:
yes, true... this is coming from some initial step where I was hiding for some reason I don't remember :)